include/ocpn_plugin.h

```diff
@@ -168,6 +168,7 @@ class PlugIn_Position_Fix_Ex {
   double Var;  // Variation, typically from RMC message
   double Hdm;
   double Hdt;
+  // The time obtained from the most recent GNSS message, or the system time if the GNSS watchdog has expired.
```

Comment:
@sebastien-rosset 

Unfortunately, this comment will not be part of the [API docs](http://opencpn.github.io/OpenCPN/api-docs/) although it IMO  should. You might want to check out the [Doxygen syntax](https://www.doxygen.nl/manual/docblocks.html)

Of course, this is an overall remark to how to comment. Yes, old code mostly does not do this. But we want to become better over time.

Reply:
Thanks. I've updated the comments